plugins/CoreConsole/Commands/GeneratePluginBase.php

```diff
@@ -397,4 +399,10 @@ private function replaceContent($replace, $contentToReplace)
         return $contentToReplace;
     }
 
+    protected function throwErrorIfNotGitInstalled()
```

Comment:
instead of calling this method in every class' execute, you could extend the `initialize` method in this base class to check that. It should be called automatically before `execute`

Reply:
I wanted to do that, but there are commands (or at least one, the `development:enable`) where we don't want to throw it. So either I create exception or call one-by-one. But TBH I haven't tried the `initalize`, I used `__construct`. I can give it a go.

Reply:
Thanks for the tip @sgiehl, I couldn't see that `initialize` method. It works, I updated the PR